sql/expression/comparison.go

```diff
@@ -3,73 +3,165 @@ package expression
 import (
 	"regexp"
 
+	errors "gopkg.in/src-d/go-errors.v0"
```

Comment:
we should be using errors v1

Reply:
Should I change it across all the repo?

Reply:
Change only the added ones, I opened an issue for the rest: https://github.com/src-d/go-mysql-server/issues/115